plugin/pom.xml

```diff
@@ -3,39 +3,34 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>com.googlecode.gwtquery.plugins</groupId>
-        <artifactId>draggable</artifactId>
+        <groupId>com.arcbees.gquery</groupId>
+        <artifactId>draggable-plugin</artifactId>
         <version>1.1.0-SNAPSHOT</version>
     </parent>
 
     <name>Draggable plugin for gQuery</name>
-    <artifactId>draggable-plugin</artifactId>
+    <artifactId>draggable</artifactId>
```

Comment:
Yeah I based it from the elastic plugin, which was `elastic-plugin` for the parent and `elastic` for the plugin itself